engines/python/setup/djl_python/tensorrt_llm.py

```diff
@@ -163,4 +218,7 @@ def handle(inputs: Input) -> Output:
         # initialization request
         return None
 
-    return _service.inference(inputs)
+    if _service.is_rolling_batch_enabled:
+        return _service.rolling_batch_inference(inputs)
```

Comment:
I like this much better, i've wanted to do this for the other handlers as well to clean up the inference methods.